test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -59,6 +61,100 @@ Describe "Invoke-Item basic tests" -Tags "CI" {
         }
         Get-Content $redirectFile -Raw | Should Match "usage: ping"
     }
+
+    Context "Invoke a folder" {
+        BeforeAll {
+            $supportedEnvironment = $true
+            if ($IsLinux)
+            {
+                $appFolder = "$HOME/.local/share/applications"
+                if (Test-Path $appFolder)
+                {
+                    $mimeDefault = xdg-mime query default inode/directory
+                    Remove-Item $HOME/InvokeItemTest.Success -Force -ErrorAction SilentlyContinue
+                    Set-Content -Path "$appFolder/InvokeItemTest.desktop" -Force -Value @"
+[Desktop Entry]
+Version=1.0
+Name=InvokeItemTest
+Comment=Validate Invoke-Item for directory
+Exec=/bin/sh -c 'echo %u > ~/InvokeItemTest.Success'
+Icon=utilities-terminal
+Terminal=true
+Type=Application
+Categories=Application;
+"@
+                    xdg-mime default InvokeItemTest.desktop inode/directory
+                }
+                else
+                {
+                    $supportedEnvironment = $false
```

Comment:
Maybe I merged too fast :) I guess this can happen on a Linux without a desktop.